data/json/overmap/multitile_city_buildings.json

```
@@ -47,6 +47,23 @@
       { "point": [ 1, 0, 2 ], "overmap": "2StoryModern04_roof_2_north" }
     ]
   },
+  {
+    "type": "city_building",
+    "id": "deserter_city",
+    "locations": [ "land" ],
+    "overmaps": [
+      { "point": [ 0, 0, 0 ], "overmap": "deserter_city_gate_north" },
+      { "point": [ 1, 0, 0 ], "overmap": "deserter_city_park_north" },
+      { "point": [ 0, 1, 0 ], "overmap": "deserter_city_office_1f_north" },
+      { "point": [ 0, 1, 1 ], "overmap": "deserter_city_office_2f_north" },
+      { "point": [ 0, 1, 2 ], "overmap": "deserter_city_office_3f_north" },
+      { "point": [ 0, 1, 3 ], "overmap": "deserter_city_office_roof_north" },
+      { "point": [ 1, 1, 0 ], "overmap": "deserter_city_office_1fb_north" },
+      { "point": [ 1, 1, 1 ], "overmap": "deserter_city_office_2fb_north" },
+      { "point": [ 1, 1, 2 ], "overmap": "deserter_city_office_3fb_north" },
+      { "point": [ 1, 1, 3 ], "overmap": "deserter_city_office_roofb_north" }
+    ]
```

```suggestion
    ],
    "flags": [ "GLOBALLY_UNIQUE" ]
```
It did in my testing it would just want to go here not on the OMTs

---

That worked, thank you!